gandi/cli/modules/paas.py

```diff
@@ -29,6 +32,70 @@ def type_list(cls, options=None):
         return cls.safe_call('paas.type.list', options)
 
     @classmethod
+    def attach(cls, name, vhost, remote_name):
+        """
+        Attach an instance to a remote from the local
+        repository.
+        """
+        remote_url = cls.git_remote(name, vhost)
+
+        ret = cls.execute('git remote add %s %s' % (remote_name, remote_url,))
+
+        if ret:
+            cls.echo('Adding remote `%s` to your local git repository.'
+                     % (remote_name))
+            cls.echo('Use `git push %s master` to push your code to the '
+                     'instance.' % (remote_name))
```

Comment:
yes indeed I read it like that 'Use `git push %s master` to push your code to the instance: %s' thus using name instead of remote_name